app/components/case_study_component/case_study_data.yml

```diff
@@ -0,0 +1,11 @@
+transDim:
```

Comment:
this is a nice feature of the components I hadn't seen, an extra place to store content